src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -61,6 +64,88 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
         };
     }
 
+    @computed get mutationRateSummary() {
+        if (this.props.store.gene.result &&
+            this.props.store.patientIds.result &&
+            this.props.store.mutationData.result &&
+            this.props.store.mskImpactGermlineConsentedPatientIds.result) {
+
+            const germlineMutationRate = this.germlineMutationRate(this.props.store.mutationData.result,
+                                                                   this.props.store.patientIds.result,
+                                                                   this.props.store.mskImpactGermlineConsentedPatientIds.result)
+            const germlineMutationRateElement:JSX.Element|null =
+                (this.props.store.mskImpactGermlineConsentedPatientIds.result.length
```

Comment:
@alisman Only impact study uses this consented patients clinical attribute. Everything else that has germline mutations should show germline mutation rate under the assumption that all patients in the study had germline screening.